scripts/prepare_lima.py

```diff
@@ -98,7 +98,7 @@ def prepare(
     torch.save(test_set, destination_path / "test.pt")
 
 
-def format_dataset(dataset_partition, include_multi_turn_conversations):
+def format_dataset(dataset_partition: dict, include_multi_turn_conversations: bool) -> List[Dict]:
```

Comment:
Why do you use dict and Dict together?

Reply:
Wow, an eagle eye you have 😄.
My excuse is a fast typing. But it's a vague one, I know.

Fixed.